common/script_values/99_steward_values.txt

```diff
@@ -266,24 +266,24 @@ promote_culture_communal_identity_bonus = {
 		multiply = 0.50
 	}
 }
-
-promote_culture_east_settling_bonus = {
+#Warcraft
+promote_culture_kalimdor_settling_bonus = {
 	value = 0
 	if = {
 		limit = {
 			scope:councillor.liege.culture = {
-				has_innovation = innovation_east_settling
+				has_innovation = innovation_kalimdor_settling
 			}
 		}
 		add = steward_promote_culture_base_total
 		multiply = 0.15
 	}
 }
-promote_culture_epidemic_bonus = {
```

Comment:
You haven't removed it thought.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113430155-bb5d0d00-93ea-11eb-992a-22a5def8fac4.png)

</details>

Reply:
I left it for the the future tenant.